Makefile

```
@@ -1,219 +1,215 @@
-	bash scripts/test_comment_ratio.sh
-
-test: test-unit
-
-e2e: test-e2e
-
-# Live integration test with real LLM (requires running Ollama)
-# Tests REPL mechanics against a real model. LLM-dependent tests SKIP gracefully.
-live: all
-	@bash e2e/test_live.sh $(BUILD_DIR)/llama-cli
+run: start ## Alias for start
+r: run
```

checkmake (0.2.2)

[warning] Target "r" should be declared PHONY.

(phonydeclared)